eng/common/scripts/common.ps1

```diff
@@ -53,3 +50,7 @@ $GetOnboardedDocsMsPackagesFn = "Get-${Language}-OnboardedDocsMsPackages"
 $GetDocsMsTocDataFn = "Get-${Language}-DocsMsTocData"
 $GetDocsMsTocChildrenForManagementPackagesFn = "Get-${Language}-DocsMsTocChildrenForManagementPackages"
 $UpdateDocsMsTocFn = "Get-${Language}-UpdatedDocsMsToc"
+$GetLanguageDisplayNameFn = "Get-${Language}-DisplayName"
```

Comment:
Do we need separate functions for each thing here as opposed to having one function that returns all the properties?

Reply:
No, use variable directly